abiquo/client.py

```diff
@@ -104,6 +107,25 @@ def _find_or_raise(self, key, ex):
             except:
                 raise ex
 
+    def refresh(self):
+        if self._extract_link('edit') is not None:
+            self.follow('edit').get()
+        else:
+            self.follow('self').get()
+
+    def put(self, data=data):
```

Comment:
Data has no value here. A default value should be provided. Anyway, should be better assume the current object is already updated, and just have a `put(self)`, and manually get the data from the object?